src/fragrance_rater/api/recommendations.py

```
@@ -0,0 +1,276 @@
+    session: Annotated[AsyncSession, Depends(get_db)],
+) -> RecommendationService:
+    """Dependency to get RecommendationService instance."""
+    return RecommendationService(session)
+
+
+@router.get("/{reviewer_id}", response_model=RecommendationListResponse)
+async def get_recommendations(
+    reviewer_id: str,
+    service: Annotated[RecommendationService, Depends(get_recommendation_service)],
+    limit: int = Query(10, ge=1, le=50, description="Maximum recommendations"),
+    exclude_rated: bool = Query(
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)